src/azure-cli-core/azure/cli/core/auth/identity.py

```diff
@@ -29,6 +29,9 @@
 AZURE_CLIENT_ID = "AZURE_CLIENT_ID"
 AZURE_CLIENT_SECRET = "AZURE_CLIENT_SECRET"
 
+WAM_PROMPT = (
+    "Please select the account you want to log in with. "
+    "For more information on login with Azure CLI, see https://go.microsoft.com/fwlink/?linkid=2271136")
```

Comment:
The message we had in the Teams chat was

```suggestion
    "For more information about login with Azure CLI go to https://go.microsoft.com/fwlink/?linkid=2271136")
```

I reworded it a little bit in order to make it consistent with other messages.

Reply:
Adding a comma before `go to` could improve the readability.

Reply:
The text shown in green is the original message. I am not going to use it.

Reply:
@Jiasli, Generally, you don't want to end a sentence with a preposition.  In docs, they also don't like us to use "please", and we are instructed to use "see", although "go to" works fine for me. 

What's really happening here?
1. Customers are selecting an account to use at login
2. Customers are setting their active subscription
3. We are providing a link if someone doesn't understand these concepts.

How about ...

WAM_PROMPT = (
    **"Select an Azure account to use for login."**
_/or/ "Select an account to set as your active subscription."
/or/ "Select an account to use at login and to set as your active subscription."_

   **For more information on login with Azure CLI, see https://go.microsoft.com/fwlink/?linkid=2271136")**
   _"For more information on logging into Azure using the Azure CLI, see ..."_
 
Italics and bold are only added to differentiate suggested edits.

Reply:
> Generally, you don't want to end a sentence with a preposition.

This is an interesting topic but controversial: https://www.scribbr.com/parts-of-speech/ending-a-sentence-with-a-preposition/